src/xdocs/writingchecks.xml

```diff
@@ -86,37 +86,39 @@
 
       <p>
         Every Java Program is structured into files, and each of these
-        files has a certain structure. For example, if there is a
-        package statement then it is the first line of the file that is
-        not comment or whitespace. After the package statement comes a
-        list of import statements, which is followed by a class or
-        interface definition, and so on.
+        files has a certain structure. For example, a package statement
+        is always the first line of a file that is not comment or whitespace.
+        After the package statement comes a list of import statements, which
+        is followed by a class or interface definition, and so on.
       </p>
       <p>
         If you have ever read an introductory level Java book you probably
-        knew all of the above. And if you have studied computer science,
+        knew all the above. And if you have studied computer science,
         you probably also know that the rules that specify the Java language
-        can be formally specified using a grammar (statement is simplified
-        for didactic purposes).
+        can be formally specified using a grammar - though this statement is
+        simplified for didactic purposes).
```

Comment:
Don't understand swapping the starting `(` for a `-` and then leaving in the ending `)`.

Reply:
Left the ending `)` on accident, fixed.

Reply:
Changed to comma